benchmarks/build.gradle

```diff
@@ -52,6 +30,8 @@ shadowJar {
     relocate 'org.eclipse.jetty', 'ai.h2o.org.eclipse.jetty'
     relocate 'org.eclipse.jetty.orbit', 'ai.h2o.org.eclipse.jetty.orbit'
     relocate 'com.google.common', 'ai.h2o.com.google.common'
+
+    exclude 'www/flow/packs/test-*/**'
```

Comment:
I would create JIRA on H2O project, just to let folks know this can be removed

Reply:
Sounds good to me. @michalkurka WDYT about having the testing flows completely separated from `h2o-docs` examples?

Reply:
I think that can be done :)